ast_tools/passes/ssa.py

```diff
@@ -176,6 +178,7 @@ class SingleReturn(InsertStatementsVisitor):
     def __init__(self,
             env: tp.Mapping[str, tp.Any],
             names_to_attr: tp.Mapping[str, cst.Attribute],
+            symbol_table_skip_targets: tp.AbstractSet[str],
```

Comment:
Is this annotation accurate? Looks like you are adding name below not strs

Reply:
Also slight preference for building a new set here instead of mutating and existing one (as I do with `added_names`) then merging where appropriate.

Reply:
(preference because it makes debugging the individual components easier -- who is adding this erroneous name to the symbol_table_skip_targets set? is easier to check if each part of the set is built individually and not through mutation.  But not a big deal)

Reply:
Ah yes, I originally was storing str, but changed to Name/Attribute since i wanted to also handle the attribute mapping, so it should be cst node